tests/functional/testplan/testing/fixtures/base/passing/report.py

```diff
@@ -1,4 +1,4 @@
-from testplan.report.testing import TestReport, TestGroupReport
+from testplan.report import TestReport, TestGroupReport
```

Comment:
A lot of unrelated changes being made... it's fine for now but in future could you split out such cosmetic changes into a separate PR?